docs/getting-started.md

```diff
@@ -0,0 +1,181 @@
+# Getting Started
+
+## Prerequisites
+
+- [Apex CLI][apex]: A project template and code generation tool.
```

Comment:
Should we add Deno to this list given its required by Apex and they'll eventually need it for any TS stuff?